jobs/build/build-sync/Jenkinsfile

```
@@ -0,0 +1,113 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    def commonlib = buildlib.commonlib
+
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '360'
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    commonlib.mockParam(),
+		    commonlib.ocpVersionParam('BUILD_VERSION', '4'),
+                ]
+            ],
+            disableConcurrentBuilds(),
+            timestamps(),
+        ]
+    )
+
+    echo "Initializing ${params.BUILD_VERSION} sync: #${currentBuild.number}"
+
+    // doozer_working must be in WORKSPACE in order to have artifacts archived
+    def mirrorWorking = "${WORKSPACE}/MIRROR_working"
```

```suggestion
    def mirrorWorking = "${env.WORKSPACE}/MIRROR_working"
```
if you don't mind...